libpod/container_config.go

```diff
@@ -376,7 +376,7 @@ type ContainerMiscConfig struct {
 	// SdNotifyMode tells libpod what to do with a NOTIFY_SOCKET if passed
 	SdNotifyMode string `json:"sdnotifyMode,omitempty"`
 	// Systemd tells libpod to setup the container in systemd mode
-	Systemd bool `json:"systemd"`
+	Systemd bool `json:"use_systemd"`
```

Comment:
^ ditto

Reply:
This is my biggest concern but needs to be done or else unmarshalling is not possible because if `json.Unmarshal` finds two fields with the same tag but different types: the whole process blows up...

Reply:
Yeah we cannot change this without breaking existing containers

Reply:
okay well is there any way to ignore errors in `json.Unmarshal`? if not I guess I will need to find another way to do this @ashley-cui @Luap99